pyproject.toml

```diff
@@ -1,59 +1,68 @@
-[tool.poetry]
+[project]
 name = "poetry"
 version = "2.0.0"
 description = "Python dependency management and packaging made easy."
-authors = ["Sébastien Eustace <sebastien@eustace.io>"]
+requires-python = ">=3.9,<4.0"
+dependencies = [
```

Comment:
I dislike this more every time i see it :sob: Part of me wants to keep the old style deps for the Poetry project itself. But, we should be good ecosystem members. **sigh**.

Reply:
Same here. We could declare `dependencies` as `dynamic` and keep it. (This does not trigger a deprecation warning.) However, this is intended more for internal projects with custom sources. I agree with _"But, we should be good ecosystem members."_

Reply:
I mean, dynamic defeats the static parsing advantage. I do not think the `dynamic` should only be promoted for the internal project cases, it should really be first class citizens too (at least from Poetry's perspective). Personally, most of my projects will likely retain the Poetry syntax.